sql/besieged_strongholds.sql

```diff
@@ -0,0 +1,51 @@
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+-- ----------------------------
+-- Table structure for `besieged_strongholds`
+-- ----------------------------
+
+DROP TABLE IF EXISTS `besieged_strongholds`;
+CREATE TABLE `besieged_strongholds`
+(
+    `stronghold_id`           tinyint(2) NOT NULL DEFAULT 0,
+    `orders`                  tinyint(2) unsigned NOT NULL DEFAULT 0,
+    `stronghold_level`        tinyint(2) unsigned NOT NULL DEFAULT 0,
+    `forces`                  tinyint(3) unsigned NOT NULL DEFAULT 0,
+    `mirrors`                 tinyint(3) unsigned NOT NULL DEFAULT 0,
+    `prisoners`               tinyint(3) unsigned NOT NULL DEFAULT 0,
+    `mirror_destroyed`        tinyint(1) unsigned NOT NULL DEFAULT 0,
+    `owns_astral_candescence` tinyint(1) unsigned NOT NULL DEFAULT 0,
+    PRIMARY KEY (`stronghold_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+LOCK
+TABLES `besieged_strongholds` WRITE;
+/*!40000 ALTER TABLE `besieged_strongholds` DISABLE KEYS */;
+
+--
+-- Dumping data for table `besieged_strongholds`
+--
+
+INSERT INTO `besieged_strongholds` (`stronghold_id`, `orders`, `stronghold_level`, `forces`, `mirrors`, `prisoners`,
+                                    `mirror_destroyed`, `owns_astral_candescence`)
+VALUES (0, 0, 0, 0, 0, 0, 0, 1),
+       (1, 0, 0, 0, 0, 0, 0, 0),
+       (2, 0, 0, 0, 0, 0, 0, 0),
+       (3, 0, 0, 0, 0, 0, 0, 0);
+
+/*!40000 ALTER TABLE `besieged_strongholds` ENABLE KEYS */;
+UNLOCK
```

Comment:
UNLOCK TABLES; is a single statement, on a single line please, same with LOCK TABLES above

Reply:
Done